Shark Cage/createACLinNewDesktop/createACLinNewDesktop/testMain.cpp

```diff
@@ -1,11 +1,43 @@
 #include "stdafx.h"
 
 #include <Windows.h>
+#include <LMaccess.h>
+#include <lmerr.h>
 #include "util.h"
 
 int main() {
-	HANDLE handle;
-	PSID sid= 0;
-	util::getModifiedToken(sid, handle);
+	LPWSTR  group_name = L"Dummy_group_testing_token_mod";
+	SID_NAME_USE accountType;

```

Comment:
In names for variables, you often switch between `fooBar` and `foo_bar`. We discussed to use in the rest of the SharkCage implementation the version with the underscore. This is an API so I think you can freely choose but in my opinion, it should be consistent?

Reply:
Good idea 👍  Fixed